.github/workflows/deploy-queue.yml

```diff
@@ -15,6 +15,7 @@ permissions:
 
 env:
   GHCR_REPO: ghcr.io/neondatabase/dev-actions
+  DEPLOY_QUEUE_DATABASE_URL: ${{ vars.DEPLOY_QUEUE_DATABASE_URL }}
```

Comment:
If this is the database we define below, why a var? If it changes, we need to change it below anyway.

If this is another database, one that's publicly accessible, it should be a secret since it contains credentials.

Having neon + branching here would be cool for the future, as it'd guarantee that we can migrate the production database with new migrations we add here. That part is something I'd move to a follow-up PR though, local only is good enough for now.